src/legacy/server/sample_data/usage/collector_fetch.ts

```diff
@@ -0,0 +1,132 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { get } from 'lodash';
+import moment from 'moment';
+
+interface SearchHit {
+  _id: string;
+  _source: {
+    'sample-data-telemetry': {
+      installCount?: number;
+      unInstallCount?: number;
+    };
+    updated_at: Date;
+  };
+}
+
+interface TelemetryResponse {
+  installed: string[];
+  uninstalled: string[];
+  last_install_date: moment.Moment | null;
+  last_install_set: string | null;
+  last_uninstall_date: moment.Moment | null;
+  last_uninstall_set: string | null;
+}
+
+export function fetchProvider(index: string) {
+  return async (callCluster: any) => {
+    const response = await callCluster('search', {
+      index,
+      body: {
+        query: { term: { type: { value: 'sample-data-telemetry' } } },
+        _source: { includes: ['sample-data-telemetry', 'type', 'updated_at'] },
+      },
+      filter_path: 'hits.hits._id,hits.hits._source',
+      ignore: [404],
+    });
```

Comment:
What's the reason behind using the search API directly vs. using the SavedObjectClient?

Reply:
The collector needs full access to `.kibana`, whereas the saved objects client is space-aware. 

Since these collections are happening in the server background, there is no user request context to initialize a saved objects client.